openwisp_monitoring/monitoring/charts.py

```diff
@@ -161,7 +161,10 @@
         'order': 260,
         'query': {
             'influxdb': (
-                "SELECT 100 * MEAN(load_1) AS CPU_load FROM {key} WHERE "
+                "SELECT 100 * MEAN(load_1) / MEAN(cpus) AS Average_1m, "
+                "100 * MEAN(load_5) / MEAN(cpus) AS Average_5m, "
+                "100 * MEAN(load_15) / MEAN(cpus) AS Average_15m "
+                "FROM {key} WHERE "
```

Comment:
why this change? I think we should have only 1 line in the chart as it was before.